src/components/Billboard.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react'
 import Script from 'react-load-script'
 import headshot from '../img/content/supercropped.jpg'
-import PageContent from './PageContent'
+import PageUnderNavbar from './PageUnderNavBar'
+
+import styles from './Billboard.module.css'
 
 export default class Billboard extends React.Component {
+  constructor(props) {
```

Comment:
this change is related to the action network widget. instead of having a new random number on every `render`, we have it every time the component is initialized which should happen when user navigates back to the page